### ISSUE TITLE
[Tooltip] Support trigger on focus (WCAG 2.1 1.4.13 Content on Hover or Focus)

### ISSUE THREAD

**Issue by sgraczyk** [livechat/design-system#350]
## Description
Tooltip should cover [WCAG 2.1 rule](https://www.w3.org/TR/WCAG21/#content-on-hover-or-focus). Currently, we're missing the trigger on the focus event.

## Design
n/a

**Comment by VadymBezpalko** [comment]
This will be fixed with https://github.com/livechat/design-system/pull/855
